### ISSUE TITLE
Combining flip-angle maps from different DREAM voltages

### ISSUE THREAD

**Issue by Kyota-exe** [spinal-cord-7t/coil-qc-code#12]
In PR #9, it was discovered that the scanner already outputs both the FA and RefV maps for DREAM scans in [this comment](https://github.com/spinal-cord-7t/coil-qc-code/pull/9#issuecomment-1917769653), and that we can simply use those for the dataset. However, it was stated earlier in the discussion [in this comment](https://github.com/spinal-cord-7t/coil-qc-code/pull/9#issuecomment-1913190172) that the FA maps from different voltages should be combined to minimize bias.

At the moment, the `bids_conversion.py` script simply takes the DREAM scans (FA map and RefV) from the optimal reference voltage (x1.0), instead of combining different voltages into a single FA map. Is this an issue? Should we modify the script so that it combines the DREAM scans?

**Comment by evaalonsoortiz** [comment]
No, it is better if the notebook retrieves all 3 scans (0.66*refV, refV, 1.5refV) and combines them.

**Comment by Kyota-exe** [comment]
I apologize if this is obvious, but how would we go about combining these scans? This is something that is not clear to me, and I would appreciate some clarification; my understanding was that we would combine all 3 FAmap NIFTI scans into a single NIFTI file, and put this in the dataset. I am beginning to understand now though that you may have meant that we should simply take the average of the data we get in all 3 voltages when we process the data in the analysis code? So we don't actually combine the scans themselves, but the results acquired from the scans...?

Assuming you do mean that we should combine the 3 NIFTI scans from different voltages into a single NIFTI scan: As you recently stated in [this comment from the original bids_conversion.py PR](https://github.com/spinal-cord-7t/coil-qc-code/pull/9#issuecomment-1952548438), the 0.66x and 1.5x scans are not currently included in the dataset.  Should I include them all in the dataset and do the combining in the analysis code, or handle this "combining" in the `bids_conversion.py` script and have it already be combined in the OpenNeuro dataset? For the same reason @jcohenadad recommended we put the processing code to generate the FAmap DREAM scans in the analysis code instead of in a pre-processing script [here](https://github.com/spinal-cord-7t/coil-qc-code/pull/9#issuecomment-1913232044), my vote would be to put it in the analysis script. 

If we put the FAmap DREAM scans for all 3 voltages in the OpenNeuro dataset however, how should we name these files to be BIDS compliant? The 1.0x FAmap scan is currently named `sub-SUBJECT_acq-famp_TB1DREAM`, so maybe `sub-SUBJECT_acq-famp-0.66_TB1DREAM` and `sub-SUBJECT_acq-famp-1.5_TB1DREAM`? Maybe @jcohenadad can pitch in on this?

**Comment by evaalonsoortiz** [comment]
The 3 DREAM scans should be part of the OpenNeuro dataset. We will combine them into one scan using the approach described here https://github.com/spinal-cord-7t/coil-qc-code/issues/7#issuecomment-1939582828 in the analysis script.

I'm not sure about the labeling, and what is and isn't accepted. I'll try to see if I can find examples.

**Comment by evaalonsoortiz** [comment]
Based on the acq-<label> entry: https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#acq I think what you proposed is ok.

**Comment by evaalonsoortiz** [comment]
I'm not sure yet if we need them or not. It might be that we have all the info we need from the flip angle maps, I'll consult with others. In the meantime add them, and we can remove them later if we don't need them.

**Comment by Kyota-exe** [comment]
This has been handled by 8522cc5. Can we close this issue?
